mimic/model/clb_objects.py

```diff
@@ -6,6 +6,7 @@
 from twisted.python import log
 from characteristic import attributes, Attribute
 from mimic.canned_responses.loadbalancer import (load_balancer_example,
+                                                 _prep_for_list,
```

Comment:
`_prep_for_list` belongs inside the `model/clb_objects.py` as against the `canned_responses`, as its converting the response object to the format required for LIST call.